.travis.yml

```diff
@@ -64,6 +72,8 @@ script:
   - case "$BUILD" in
       "BAZEL")
         bazel test ... --javacopt=-Werror --javacopt=-Xlint:all --javacopt=-Xlint:-cast --javacopt=-Xlint:-deprecation --javacopt=-Xlint:-try --javacopt=-Xlint:-rawtypes --javacopt=-Xlint:-serial --javacopt=-Xlint:-unchecked --config=error_prone_warnings --config=error_prone_experimental_warnings ;;
+      "GRADLE")
+        ./gradlew check :instrumentation-all:jacocoTestReport --stacktrace ;;
```

Comment:
Because here we want to have jacoco for code coverage. https://github.com/jacoco/jacoco

https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_tasks
assemble & check are the minimum to make sure code compiles & passes tests (and they are part of the java plugin).

The :instrumentation-all:jacocoTestReport makes sure that checkstyle, and all the plugins that we have run and produce the reports (if any).